src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -58,7 +59,7 @@ export const NewAccountTosScene = (props: Props) => {
   const terms: string[] = [
     sprintf(s.strings.terms_one, appName),
     s.strings.terms_two,
-    sprintf(s.strings.terms_three, appName),
+    ...(isLightAccount ? [] : [sprintf(s.strings.terms_three, appName)]),
```

Comment:
The `isLightAccount` variable is not defined here. Did this line accidentally come in from a future commit?